dashboards/Insight/Home_Dashboard.json

```diff
@@ -1465,7 +1465,7 @@
           "mappings": [],
           "max": 100,
           "min": 60,
-          "noValue": "No CPU change in last 7 days",
+          "noValue": "CPU usage has a Delta of less than 30% as compared to previous week",
```

Comment:
sorry Anton but Delta doesnt mean anything to me :( @rasika-chivate can you help Anton to rephrase the sentence structure?

Reply:
At right side we can see "High CPU" when CPU utilisation more than 90%. At "Anomaly" we take current CPU utilisation and comparing it this utilisation one week ago. If current utilisation more, than 60% and one week ago it was more, than 30% difference we got alert at this panel.